frontend/src/Metamaps/PasteInput.js

```diff
@@ -29,12 +28,10 @@ const PasteInput = {
       if (e.dataTransfer.files.length > 0) {
         self.handleFile(e.dataTransfer.files[0], coords)
       }
-      // OMG import bookmarks 😍
+      // OMG import bookmarks 😍 (Or just text :P)
```

Comment:
Correct me if I'm wrong, but I think this particular code path only happens for bookmarks. If you verify that by putting a debugger here and triggering it without a bookmark I'll believe you, but I think paste and drag-drop trigger at different points. This idea of a "dataTransfer.items" I think only happens for bookmarks.